src/itype.h

```diff
@@ -202,29 +202,30 @@ struct islot_brewable {
     time_duration time = 0_turns;
 };
 
-struct islot_armor {
-    /**
-     * Bitfield of enum body_part
-     * TODO: document me.
-     */
-    body_part_set covers;
-    /**
-     * Whether this item can be worn on either side of the body
-     */
-    bool sided = false;
-    /**
-     * How much this item encumbers the player.
-     */
+struct random_armor_data {
```

Comment:
Agreed that `random_armor_data` is a bad name.  Maybe `armor_portion_data`?  Still not great...